src/sentry/static/sentry/app/components/charts/miniBarChart.tsx

```diff
@@ -62,10 +62,13 @@ class MiniBarChart extends React.Component<Props> {
           updated.barMinHeight = 1;
           updated.barGap = '-100%';
         }
+        set(updated, 'itemStyle.color', colors[i]);
         set(updated, 'itemStyle.opacity', 0.6);
         set(updated, 'itemStyle.emphasis.opacity', 1.0);
         if (emphasisColors && emphasisColors[i]) {
           set(updated, 'itemStyle.emphasis.color', emphasisColors[i]);
+        } else {
+          set(updated, 'itemStyle.emphasis.color', colors[i]);
```

Comment:
Would this be easier to read if this entire if/else was a single statement? e.g.

```
set(updated, 'itemStyle.emphasis.color', emphasisColors?.[i] ?? colors[i]);
```